src/SprykerEco/Client/ArvatoRss/Zed/ArvatoRssStub.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Client\ArvatoRss\Zed;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Client\ZedRequest\Stub\ZedRequestStub;
+
+class ArvatoRssStub extends ZedRequestStub implements ArvatoRssStubInterface
+{
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
```

Comment:
Is the code below marked "yellow"? We in the core had recently to also add the interface here.